Automattic/wp-calypso
# Email stats: list unknown countries

#### Proposed Changes

* This PR fixes changes how unknown countries are shown in the email open stats

| Before | After |
|--------|------|
| ![image](https://user-images.githubusercontent.com/528287/212332213-ef30473c-cfd9-483e-9a75-0169a273916d.png) | <img width="1113" alt="CleanShot 2023-01-13 at 14 35 23@2x" src="https://user-images.githubusercontent.com/528287/212332283-ebddd8b6-eedd-4989-b10d-5b8ea7a447a9.png"> |

Visual alignment of the labels will be done in a future PR.


#### Testing Instructions

* Apply this PR
* Visit `/stats/email/open/<site>/day/<postid>?flags=newsletter/stats`
* When viewing open email stats, unknown countries should be marked as Unknown

#### Pre-merge Checklist
